python/pyspark/pandas/base.py

```diff
@@ -982,9 +982,11 @@ def all(self, axis: Axis = 0, skipna: bool = True) -> bool:
               original column labels.
 
         skipna : boolean, default True
-            Exclude NA/null values. If an entire row/column is NA and skipna is True,
+            Exclude NA values, such as None or numpy.NaN.
```

Comment:
Rephrased to clarify that NA values include None and numpy.Nan, and they behave differently when `skipna` is setFalse.